packages/react-scripts/template/README.md

```diff
@@ -1 +1 @@
-Learn about making a Progressive Web App [here](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
```

Comment:
The old README actually has 6 different links on it that were referenced by old bitly links so replacing this with a link to one of them doesn't really work.